.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+If the provided content is part of the present PR remove the # symbol.
+
+- [ ] **changelog** provided | not required because: 
+- [ ] **development PR** provided # | not required because: 
+- [ ] **framework PR** provided # | not required because: 
+- [ ] **3.6 PR** provided # | not required because: 
+- [ ] **2.28 PR** provided # | not required because: 
+- **tests**  provided | covered # | not required because: 
```

Comment:
Covered elsewhere is exceptional for tests and we shouldn't suggest that it's ok.
```suggestion
- **tests**  provided | not required because: 
```

Reply:
This was discussed in the design review and the reference PR before. In theory it is exceptional but in practise it is way more common, especially by contributions raised by us, where a lot of complicated things are spread into multiple prs. 

I will remove it so we can move forward, but I expect to see the justification of covered in the because section.

e.g
    - **tests**  not required because:  they are covered at #